python/tidy/servo_tidy/tidy.py

```diff
@@ -360,6 +351,43 @@ def find_reverse_dependencies(dependency, version, content):
 """.format(**substitutions).strip()
                 yield (1, message)
 
+        # Packages with same version from different sources
+        if (len(set(data["version"])) < len(data["version"])
+                and len(set(data["source"])) == len(data["source"])):
+            # Set recommended source
+            if r"registry+https://github.com/rust-lang/crates.io-index" in data["source"]:
+                req_source = r"registry+https://github.com/rust-lang/crates.io-index"
+            elif "github.com/servo/" in data["source"]:
```

Comment:
Enlighten me. Do we have such a case in that list? This looks for an exact match of `"github.com/servo/servo` in the `data["source"]` list, no?